Tests/TestingApps/PaywallsTester/PaywallsTester/Mocks/MockData.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  MockData.swift
+//  
+//
+//  Created by Nacho Soto on 12/14/23.
+//
+
+import Foundation
+import OSLog
+
+
+#if DEBUG
+
+public enum MockData {
+
+    public static func developer() -> DeveloperResponse {
+        return try! MockData.decode(Self.developerJson)
+    }
+
+}
+
+
+extension MockData {
+
+    static let developerJson = """
+    {
+        "apps": [
+            {
+                "api_key": "",
+                "bundle_id": "com.revenuecat.sampleapp",
+                "enable_price_experiments": false,
+                "feature_overrides": [],
+                "id": "asdasdsad",
+                "name": "Main RevenueCat Sample App",
+                "on_restore": null,
+                "owner_email": "nacho@revenuecat.com"
+            }
+        ],
+        "current_plan": "pro",
+        "distinct_id": "asasdasd",
+        "email": "nacho@revenuecat.com",
+        "email_verified": true,
+        "name": "Ignacio Soto Bustos"
+    }
+    """
+
+}
+
+// MARK: -
+
+
+
+extension MockData {
```

Comment:
total nitpick, but we have a style guide here: https://github.com/RevenueCat/purchases-ios/blob/main/Contributing/SwiftStyleGuide.swift, we should keep the line breaks somewhat in check